vpr/src/route/check_route.cpp

```diff
@@ -472,26 +472,14 @@ static int chanx_chany_adjacent(int chanx_node, int chany_node) {
     /* Returns 1 if the specified CHANX and CHANY nodes are adjacent, 0         *
```

Comment:
Separate enhancement, but it would be good if you check if this routine should really return a bool (and the calling code should check that).

Reply:
I believe this is correct. The code that uses this function is counting the number of adjacent nodes, so it wants a 1 or 0.

`num_adj += chanx_chany_adjacent(from_node, to_node);`